core/domain/exp_domain.py

```diff
@@ -2460,6 +2525,35 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    def to_proto(self):
```

Comment:
Is the actual proto structure verified in a test? That's actually really important. We should make sure that for this & all other to_proto functions that each field is verified in tests to ensure the proto is being populated correctly.

Reply:
Added test in every interaction package.